kivy/uix/label.py

```diff
@@ -812,3 +813,38 @@ def print_it(instance, value):
     :attr:`strip` is a :class:`~kivy.properties.BooleanProperty` and
     defaults to False.
     '''
+
+    font_hinting = OptionProperty(
+        'none', options=['normal', 'light', 'mono', 'none'])
+    '''What hinting option to use for font rendering.
+    Can be `'normal'`, `'light'`, `'mono'`, `'none'`
+
+    .. note::
+        This feature requires a SDL2 window provider and is currently only
+        supported on desktop platforms.
+
+    :attr:`font_hinting` is an :class:`~kivy.properties.OptionProperty` and
+    defaults to `'none'`.
+    '''
+
+    font_kerning = BooleanProperty(False)
+    '''Whether kerning is enable for font rendering
+
+    .. note::
+        This feature requires a SDL2 window provider and is currently only
+        supported on desktop platforms.
+
```

Comment:
Same as above, remove the warning about desktops and include ..versionadded::1.9.2